crm_timesheet/tests/test_account_analytic_line.py

```diff
@@ -0,0 +1,32 @@
+# Copyright 2017 Jairo Llopis <jairo.llopis@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp.tests.common import SavepointCase
```

Comment:
Wouldn't it be `odoo.tests.common`
:wink:

Reply:
@oihane sure! 

Done.